CMakeLists.txt

```diff
@@ -70,6 +75,17 @@ if (USE_OPENMP)
   endif ()
 endif ()
 
+set(LIBCMAES_DEPS "")
+
+if (USE_TBB)
+  find_package (TBB)
+  if (TBB_FOUND)
+    add_definitions (-DUSE_TBB)
+    include_directories (${TBB_INCLUDE_DIRS})
+    # should use ${TBB_LIBRARIES} but it is not working properly
+    set (LIBCMAES_DEPS "tbb")

```

Comment:
why cannot you use TBB_LIBRARIES ?

Reply:
> why cannot you use TBB_LIBRARIES ?

To be honest, I do not remember. But the only TBB library is "tbb" so there should not be any issue.

Reply:
It can be tbb_static too, see windows binaries.

Reply:
> It can be tbb_static too, see windows binaries.

I do not have a windows machine to test. You're free to test it and make the appropriate changes. Thanks!

Reply:
It should be TBB_LIBRARIES.

Reply:
This should be fixed now..